src/badge/index.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from 'enzyme'
+import Badge from 'govuk-react-components/badge';
+
+describe('The Badge Component', () => {
+  it('should render children that are passed in', () => {
+    expect(
+      render(<Badge>Sample Text</Badge>)
+    ).toMatchSnapshot()
+  });
+
+  it('should append classNames when attribute is passed in', () => {
+    expect(
+      render(<Badge className="my-additional-classname">Sample Text</Badge>)
+    ).toMatchSnapshot();
+  });
+
+});
```

Comment:
This file needs to be in a `__test__` directory to not get linted (and therefore not to fail on undefined global vars)

Reply:
Should have fixed that: 
https://github.com/lennym/govuk-react-components/pull/23/files#diff-1b0fd5322de42e11c8786543f74c7c41R6